content/faq/32_how-to-know-if-my-registration-is-successful/index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: how will I know that my registration is successful? Am I in first 50?
```

Comment:
remove the first 50 part it will become generic.

Reply:
Removed.